### PR TITLE
S0-613 - basic styling for supplier account and profile builder

### PR DESCRIPTION
**Pull request by adohall**
- these are essentially the changes made for SO-547, a ticket which has since been deleted because the re-styling work for the profile builder was shifted to SO-613 (this PR); SO-548 (not yet started) will consist mainly of functional changes to the profile builder; ignore references to SO-548 in my commits below - they should read SO-613;
- refactoring of several components was required to allow the older (registration) styling to co-exist unchanged beside the new phase 2 (wayfinder) styling

### CODE REVIEW

File: app/templates/components/dds/multi-text.hbs [hunk 1]
```diff
@@ -1,35 +1,25 @@
...
-        thin=thin
-        noLabel=true
-        wide=true
-        apiError=(get apiError index)
-        showError=showError
-      }}
-    </div>
-    <div class="clear-both"></div>
-  </p>
+  {{dds/text-input
```
Comment (armansa) [inline app/templates/components/dds/multi-text.hbs]: This is used in supplier application in addition to supplier profile, if you have checked and both look good, please ignore.

File: app/templates/components/seller-profile/capability-and-experty-edit.hbs [hunk 2]
```diff
@@ -1,86 +1,58 @@
...
-    thin=true
-    invalid=(v-get form 'methodologies' 'isInvalid')
-    errorMessage=(v-get form 'methodologies' 'message')
-    apiError=form.apiErrors.methodologies
-    showError=form.showError
-  }}
-</p>
-<hr class="super-light">
+<h4>Methodologies</h4>
+<p class="description">What are the different project management methodologies used within your company that may be of interest to buyers?<</p>
```
Comment (armansa) [inline app/templates/components/seller-profile/capability-and-experty-edit.hbs]: Please remove that extra <

**Comment by armansa** [comment]
Can you please resolve the conflicts?

**Comment by armansa** [comment]
I quickly tested it on mobile view, the text boxes and text areas are not visible, and some titles are not aligned well.

File: app/templates/components/dds/multi-text.hbs (hunk 1 above)
Comment (adohall) [reply app/templates/components/dds/multi-text.hbs]: Yes I checked in supplier application (e.g. membership and awards). 'add row' still has the smaller dark blue version of the plus circle.

File: app/templates/components/seller-profile/capability-and-experty-edit.hbs (hunk 2 above)
Comment (adohall) [reply app/templates/components/seller-profile/capability-and-experty-edit.hbs]: Done

**Comment by adohall** [comment]
Fixes for narrow viewports have been done in PR for SO-544 (to prevent potential conflicts).

**Comment by adohall** [comment]
Conflicts resolved.

**Pull request closed without merging**